src/features/network/network-provider.tsx

```diff
@@ -7,39 +7,42 @@ import {
   Account,
   Base,
   Events,
-} from "@liftedinit/many-js"
-import { useNetworkStore } from "./store"
-import { useAccountsStore } from "features/accounts"
+  Tokens,
+} from "@liftedinit/many-js";
+import { useNetworkStore } from "./store";
+import { useAccountsStore } from "features/accounts";
 
 const NetworkContext = React.createContext<[Network?, Network?]>([
   undefined,
   undefined,
-])
+]);
 
 export function NetworkProvider({ children }: React.PropsWithChildren<{}>) {
-  const activeNetwork = useNetworkStore(state => state.byId.get(state.activeId))
-  const activeAccount = useAccountsStore(state =>
-    state.byId.get(state.activeId),
-  )!
+  const activeNetwork = useNetworkStore((state) =>
+    state.byId.get(state.activeId)
+  );
+  const activeAccount = useAccountsStore((state) =>
+    state.byId.get(state.activeId)
+  )!;
 
   const network = React.useMemo(() => {
-    const anonIdentity = new AnonymousIdentity()
-    const identity = activeAccount?.identity ?? anonIdentity
-    const url = activeNetwork?.url || ""
-    const queryNetwork = new Network(url, anonIdentity)
-    queryNetwork.apply([Ledger, IdStore, Account, Events, Base])
-    const cmdNetwork = new Network(url, identity)
-    cmdNetwork.apply([Ledger, IdStore, Account])
-    return [queryNetwork, cmdNetwork] as [Network, Network]
-  }, [activeNetwork, activeAccount])
+    const anonIdentity = new AnonymousIdentity();
+    const identity = activeAccount?.identity ?? anonIdentity;
+    const url = activeNetwork?.url || "";
+    const queryNetwork = new Network(url, anonIdentity);
+    queryNetwork.apply([Ledger, IdStore, Account, Events, Base, Tokens]);
+    const cmdNetwork = new Network(url, identity);
+    cmdNetwork.apply([Ledger, IdStore, Account, Tokens]);
```

Comment:
Modules have to be explicitly bound _in the client app_ to be accessed. I don't love this pattern and will work to address it in ManyJs.

Reply:
Hmm... let me think about this a little more then. My gut says, if we're trying to treeshake and not supply the whole library, we should consider something more like this:

```js
import Ledger from "@liftedinit/many/ledger";

const ledger = new Ledger(url, identity);
const info = await ledger.info();
```

Reply:
Star re-exports are properly tree shaken by webpack, but you're right.